packages/block-library/src/image/edit.js

```diff
@@ -588,21 +764,35 @@ class ImageEdit extends Component {
 								/* eslint-enable jsx-a11y/no-noninteractive-element-interactions */
 							);
 
+							// Floating a resized image can produce inaccurate `imageWidthWithinContainer`.
```

Comment:
I'm not really sure what this comment has to do with the logic which follows it. If `imageWidthWithinContainer` is producing an inaccurate value, should that be corrected from within `ImageSize` ? I don't really see how it has any impact one way or the other on how it's used here though.

Reply:
Uh, this comment seems left over from previous iterations... At some point it was using the imageWidthWithinContainer.